README.md

```diff
@@ -6,6 +6,10 @@ _Go application used as a web backend with storage for custodial wallet's xpriv
 
 The `spv-wallet-web-backend` is an HTTP server that serves as the referential backend server for a custodial web wallet for Bitcoin SV (BSV). It integrates and utilizes the `spv-wallet` component to handle various BSV-related operations, including the creation of transactions and listing incoming and outgoing transactions.
 
+## Documentation
```

Comment:
Remove header. This sentence can be part of previous paragraph

Reply:
Changed